### PR TITLE
create admin folder inside index file

### PR DESCRIPTION
**Pull request by Sakshi-Prajapat**

### CODE REVIEW

File: admin/public/logo192.png
Comment (tw-vineet) [inline admin/public/logo192.png:1]: Replace logo with your app logo

File: admin/src/App.css
Comment (tw-vineet) [inline admin/src/App.css:1]: Remove unwanted code in this file.

File: admin/src/screen/dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+  const card = [
+    {
+      number: "3456",
+      title: "hello",
+      percentage: "20%",
+      icon: (
+        <KeyboardArrowDownIcon
+          sx={{
+            height: "12px",
+            width: "12px",
```
Comment (tw-vineet) [inline admin/src/screen/dashboard/Dashboard.tsx]: Remove inline style in hole code.

File: admin/public/logo192.png
Comment (Sakshi-Prajapat) [reply admin/public/logo192.png:1]: ok

File: admin/src/screen/dashboard/Dashboard.tsx (hunk 1 above)
Comment (Sakshi-Prajapat) [reply admin/src/screen/dashboard/Dashboard.tsx]: it was removed

File: admin/src/App.css
Comment (Sakshi-Prajapat) [reply admin/src/App.css:1]: the code was removed

**Pull request merged**